### ISSUE TITLE
[BUG] CLI: local variable 'x_' referenced before assignment

### ISSUE THREAD

**Issue by PalmtopTiger** [Ousret/charset_normalizer#74]
**Describe the bug**
UnboundLocalError: local variable 'x_' referenced before assignment

**To Reproduce**
`normalizer WISHGEN.TXT`
[WISHGEN.TXT](https://github.com/Ousret/charset_normalizer/files/6875653/WISHGEN.TXT)

**Expected behavior**
The program does not crash.

**Logs**
```
Unable to identify originating encoding for "WISHGEN.TXT". Maybe try increasing maximum amount of chaos.
Traceback (most recent call last):
  File "c:\python39\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "c:\python39\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Python39\Scripts\normalizer.exe\__main__.py", line 7, in <module>
  File "c:\python39\lib\site-packages\charset_normalizer\cli\normalizer.py", line 192, in cli_detect
    ] if args.alternatives else x_[0].__dict__,
UnboundLocalError: local variable 'x_' referenced before assignment
```

**Desktop (please complete the following information):**
 - Windows 10 x64
 - Python 3.9.6 x64
 - Package version: 2.0.3

**Comment by Ousret** [comment]
Hi @PalmtopTiger 

Thanks for submitting this. Fortunately, I have found it a bit sooner, will be fixed in the next minor https://github.com/Ousret/charset_normalizer/pull/70

I am closing this, feel free to test the patch from dev-master.

Regards,

**Comment by Ousret** [comment]
Fixed in version 2.0.4
